### PR TITLE
Update russian translation

### PR DESCRIPTION
**Pull request by r57zone**
Hi, I updated russian translation a bit and added new strings of the latest changes.

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -66,7 +77,9 @@
     <string name="set_bookmark_at">Поставить закладку на</string>
     <string name="no_bookmarks">Пока еще нет закладок в этой аудиокниге.</string>
     <string name="bookmarks">Закладки</string>
+    <string name="delete">Удалить</string>
     <string name="delete_from_db">Удалить из базы данных</string>
+    <string name="delete_from_db">Удалить из библиотеки</string>
```
Comment (flackbash) [inline app/src/main/res/values-ru/strings.xml:82]: I think this should be `remove_from_library` instead of `delete_from_db`

**Comment by flackbash** [comment]
Great, thank you :) If you could just make this small adjustment then I will merge it.

**Comment by flackbash** [comment]
Could you squash all three commits into the first one (using `git rebase -i HEAD~3`) to keep the history a little cleaner, then I'll really merge it :)

**Comment by r57zone** [comment]
Seems to fixed

**Comment by flackbash** [comment]
Yes, perfect, thank you :)

**Pull request merged**